pennylane/tracker.py

```diff
@@ -100,7 +100,7 @@ def circuit(x):
         whether ``execute`` or ``batch_execute`` last performed an update.
 
         >>> def shots_info(totals, history, latest):
-        ...     if 'shots' in latest.keys():
+        ...     if 'shots' in latest:
```

Comment:
pylint caught this from inside a docstring??

Reply:
No, it caught one version of this usage in the code and I noticed (via search-in-code) that there is another in the docstring :)
I thought it might be nice to sync docstring and codebase styles :)